### ISSUE TITLE
Bug with NVDA+shift+f3

### ISSUE THREAD

**Issue by burakyuksek** [nvaccess/nvda#11770]
### Steps to reproduce:
1. Make sure you didn't type anything in the NVDA+f edit field.
2. Go to a document that you need text found for
3. Go to the end of the document
4. Press NVDA+shift+f3
5. Type the word
6. NVDA will say text not found. But when you press NVDA+shift+f3 again NVDA will be able to find the word you're looking for.
### Actual behavior:
When steps 4 and 5 are complete NVDA doesn't try to find previous result, it tries to go to the next result which fails because the currsor is at the end of the document.
### Expected behavior:
When steps 4 and 5 are applied NVDA should be able to find last occurrance
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2020.3
#### Windows version:
2004
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2020.2, issue was there
